src/main/java/com/ironz/unsafe/UnsafeAndroid.java

```diff
@@ -467,6 +522,14 @@ public void putDouble(Object obj, long offset, double newValue) {
      * @since 1.8
      */
     @TargetApi(24)
+    public void copyMemory(Object srcObj, long srcAddr, Object dstObj, long dstAddr, long bytes) {
```

Comment:
That's interesting, because it's part of the openjdk and I'm able to use this method on my device running Android o beta.